main.go

```diff
@@ -58,6 +59,10 @@ func runMain() error {
 	}
 	log.Printf("config: %+v", *conf)
 
+	if err := deleteTempFiles(conf.OutputDir); err != nil {
```

Comment:
Add a log line before this so we know what the recorder is doing. It could take a while if there's tons there for some reason.